3001-4000/LIT3529Daniel.xml

```diff
@@ -34,8 +34,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p>Biblical Book of Daniel, including the so-called deuterocanonical books with the stories about Susanna, Bel and the dragon, 
-               the Prayer of Azariah and the Three Holy Children.</p>
+            <p><ref type="work" corresp="wd:Q80115">Biblical Book of Daniel</ref>, including the so-called deuterocanonical books with 
```

Comment:
above in 1259 you use the same `wikidata ID `for the book of Daniel proper. if the use above is correct, remove `wd `here, as it only multiplies the confusion, the same ID may not be used for the two different constellations of texts